packages/webapp/pages/_app.tsx

```diff
@@ -94,6 +95,8 @@ function InternalApp({ Component, pageProps, router }: AppProps): ReactElement {
   }, [updateCookieBanner, user]);
 
   useEffect(() => {
+    initApp();
```

Comment:
Probably better to have this in its own useEffect as this one can run more than once.